init (#8215)
PennyLaneAI/pennylane: doc/releases/changelog-dev.md

<h3>Bug fixes 🐛</h3>

> * Autograph will now be run correctly on nested functions of `qml.adjoint` and `qml.ctrl`.

Author: ```suggestion
* Autograph will now be correctly applied to nested functions of `qml.adjoint` and `qml.ctrl`.
```

> * Autograph will now be correctly applied to nested functions of `qml.adjoint` and `qml.ctrl`.

Reviewer: ```suggestion
* Autograph will now be correctly applied to nested functions of :func:`~pennylane.adjoint`
  and :func:`~pennylane.ctrl`.
```
